src/services/nats.ts

```diff
@@ -55,25 +55,25 @@ export function registerPrivateHandlers(
     prefix: string,
     handlers: PrivateNatsHandler[]
 ): void {
-    handlers.map(([subject, handler, options]) => {
+    handlers.map(([subject, handler]) => {
         const fullSubject = `${prefix}.${subject}`;
         console.log(
             `[TOKENIZATION-SERVICE] Registering handler ${fullSubject}`
         );
-        handler(natsConnection.subscribe(fullSubject, options));
```

Comment:
why are we removing options? options is what allows us for instance to configure queue groups, which ensures load balancing between micro services